202/_dev/modules/fetch-paypal-carriers/index.php

```diff
@@ -1,6 +1,6 @@
 <?php
```

Comment:
Could we please remove every index.php from code :  
find . -name 'index.php' -exec git rm {} --cached \;  
Thank you.

Reply:
index.php were removed
upgrade file is not required